DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -1547,8 +1547,8 @@ function MarkBingoFailedSpecial()
         FailIfCorpseNotHeld(class'#var(prefix)TerroristCommanderCarcass', "LeoToTheBar");
         FailIfCorpseNotHeld(class'#var(prefix)PaulDentonCarcass', "PaulToTong");
         break;
-    case "06_HONGKONG_WANCHAI_MARKET":
-    case "06_HONGKONG_WANCHAI_CANAL":
+    case "06_HONGKONG_VERSALIFE":
+    case "06_HONGKONG_STORAGE":
```

Comment:
this isn't quite right for entrance rando, maybe should have a separate switch/case just for mission number